### DISCUSSION TITLE
Video Resolution (Qualities) selector or how to set MaxBitRate for m3u8 link in MediaElement

### DISCUSSION THREAD

**Discussion by vikashhd** [CommunityToolkit/Maui#2069] [General]
Currently MediaElement supports m3u8 links but video quality is auto. But to set MaxBitRate manually so that User can select video quality like 360 /480/720 so is there any way to do that ?

**Comment by ghost** [comment]
Talking to CoPilot, it would be possible to get playlist of m3u8 and get the exact video URL based on video quality. That's just an idea, need to check in real life if it would work. The library that CoPilot suggest is [globocom/m3u8](https://github.com/globocom/m3u8) ... but it's in Pyhton.
